visual-java/src/main/java/com/saucelabs/visual/VisualApi.java

```diff
@@ -520,13 +512,39 @@ private List<RegionIn> extractIgnoreList(CheckOptions options) {
     return result;
   }
 
+  private List<ElementIn> extractIgnoreElements(CheckOptions options) {
+    if (options == null) {
+      return Collections.emptyList();
+    }
+
+    List<WebElement> ignoredElements =
+            options.getIgnoreElements() == null ? Arrays.asList() : options.getIgnoreElements();
+
+    List<ElementIn> result = new ArrayList<>();
+    for (int i = 0; i < ignoredElements.size(); i++) {
+      WebElement element = ignoredElements.get(i);
+      if (validateRemoteElement(element) == null) {
+        throw new VisualApiException("options.ignoreElement[" + i + "] does not exist (yet)");
+      }
+      result.add(VisualRegion.ignoreChangesFor(element).toElementIn());
+    }
+    return result;
+  }
+
   private WebElement validate(WebElement element) {
     if (element == null || !element.isDisplayed() || element.getRect() == null) {
       return null;
     }
     return element;
   }
 
+  private WebElement validateRemoteElement(WebElement element) {
+    if (element == null || !element.isDisplayed() || !(element instanceof RemoteWebElement)) {
+      return null;
+    }
+    return element;
+  }
```

Comment:
maybe easier to read
```suggestion
    if (element != null && element instanceof RemoteWebElement && element.isDisplayed()) {
      return element;
    }
    return null;
  }
```